tests/artifacts/dataset_manager/configuration4.ini

```diff
@@ -0,0 +1,19 @@
+[GENERAL]
+dataset_path = tests/artifacts/dataset_manager/dataset_manager_test_data
+
+[SCHEDULER]
+num_runners = 4
+frequency_in_mins = 1
+
+[RUNNER]
+weights_directory = weights
+
+[BLOCKCHAIN]
+ipfs_port = 5001
+http_port = 3001
+host = 127.0.0.1
+timeout = 5
+
+[DATASET_MANAGER]
+category = default_category
+sample_fraction = 0.001
```

Comment:
The default in `questions.csv` is 0.01, so was 0.001 intentional here?

Reply:
Fixed